config/locales/en.yml

```diff
@@ -26,7 +26,7 @@
 # en:
 #   'true': 'foo'
 #
-# To learn more, please read the Rails Internationalization guide
+# For more information, please read the Rails Internationalization guide
 # available at https://guides.rubyonrails.org/i18n.html.
 
```

Comment:
All of the above was auto-generated by Rails ;-).

Reply:
I wouldn't have even touched this commented lines, but as they were already changed in this PR, I thought why not 🙈